build.gradle

```diff
@@ -95,7 +95,7 @@ dependencies {
 
     api group: 'org.elasticsearch', name: 'elasticsearch', version: '7.17.24'
 
-    implementation group: 'com.github.hmcts', name: 'et-data-model', version: '3.17.4'
+    implementation group: 'com.github.hmcts', name: 'et-data-model', version: '97cad9441c'
```

Comment:
Are you planning on merging this PR to master? If so, can this tag be changed to a released version please?

Reply:
It is the wrong link after approval of et-data-model will be merged